tools/test.py

```diff
@@ -69,4 +61,9 @@ def main(argv):
 
 
 if __name__ == '__main__':
-    sys.exit(main(sys.argv))
+    if len(sys.argv) == 2:
+        build_dir = sys.argv[1]
+    else:
+        print "Usage: tools/test.py out/debug"
+        sys.exit(1)
+    sys.exit(main(build_dir))
```

Comment:
This doesn't look right ... it invokes main() even if the build_dir argument is not present.

Reply:
Sorry nvm, I missed the sys.exit() just above it.